posthog/clickhouse/cluster.py

```diff
@@ -52,7 +52,9 @@ def result(
 
 class ConnectionInfo(NamedTuple):
     address: str
-    port: int
+
+    def make_pool(self) -> ChPool:
+        return make_ch_pool(host=self.address)
```

Comment:
Probably will fold this back into `HostInfo` at some point as it's now overkill, but not really a priority right now.